structopt-derive/src/lib.rs

```diff
@@ -500,7 +516,9 @@ fn gen_augment_clap_enum(
         fn augment_clap<'a, 'b>(
             app: ::structopt::clap::App<'a, 'b>
         ) -> ::structopt::clap::App<'a, 'b> {
-            app #app_methods #( #subcommands )* #version
+            let app = app #app_methods #version;
+            #( #subcommands )*;
+            app
```

Comment:
`#version` must go after `#subcommands` here due to another bug

Reply:
done

Reply:
There's the fancy grey `resolve Conversation` button next to such comments; you should hit it instead of wasting energy on typing "done", let's be energy-efficient!

Reply:
I thought it was for the reviewer to decide if this is resolved or not :grin:

Reply:
Don't you worry, I have the even fancier `Unresolve conversation` button